angelsbioprocessing/prototypes/technology/bio-processing-wood-paper.lua

```diff
@@ -110,36 +113,41 @@ data:extend({
       },
       {
         type = "unlock-recipe",
-        recipe = "tree-arboretum-1",
+        recipe = "bio-arboretum-1",
       },
       {
         type = "unlock-recipe",
-        recipe = "bio-arboretum-1",
+        recipe = "composter",
+      },
+      {
+        type = "unlock-recipe",
+        recipe = "solid-soil",
       },
       {
         type = "unlock-recipe",
         recipe = "tree-generator-1",
       },
       {
         type = "unlock-recipe",
-        recipe = "wood-sawing-1",
+        recipe = "tree-arboretum-1",
       },
       {
         type = "unlock-recipe",
-        recipe = "solid-saw",
+        recipe = "wood-sawing-1",
       },
       {
         type = "unlock-recipe",
-        recipe = "bio-resin-wood-reprocessing",
+        recipe = "solid-saw",
       },
     },
```

Comment:
Too many recipe unlocks for the tech window
![image](https://github.com/Arch666Angel/mods/assets/26593477/2b523e20-cc55-432c-bdf9-0a645b1e0001)

I would suggest moving `composter` and `solid-soil` onto a basic technology `composting` description `Recycling of organic left-over materials`

Tech progression:
Washing + Algae farming => Composting
Composting => Arboretum 1

At that point Farming 1 should not depend anymore on Arboretum 1, and solely on Composting, keeping farming and tree processing separate instead of depending on each other?

Reply:
![image](https://github.com/Arch666Angel/mods/assets/59639/bfc6c487-cd16-453a-b8f9-fc23a75e6aa2)